HPXMLtoOpenStudio/resources/schedules.rb

```diff
@@ -1145,6 +1275,50 @@ def self.get_begin_and_end_dates_from_monthly_array(months, year)
 
     return begin_month, begin_day, end_month, end_day
   end
+
+  def self.get_schedules_affected
+    affected_csv = File.join(File.dirname(__FILE__), 'data', 'schedules_affected.csv')
+    if not File.exist?(affected_csv)
+      fail 'Could not find schedules_affected.csv'
+    end
+
+    require 'csv'
+    schedules_affected = CSV.open(affected_csv, headers: :first_row).map(&:to_h)
```

Comment:
This is going to re-read the same file over and over for every schedule column. Can we just read it once?

Reply:
Oops, I did that in other places, but not everywhere.